site/styles/_awards.scss

```diff
@@ -3,9 +3,10 @@
 
   &.meta-awards-all {
     display: grid;
-    gap: 30px 104px;
+    gap: 28px 104px;
```

Comment:
Judging from what I see in the GoogleDoc... that horizontal gap should be 88px instead of 104px.  Am I understanding what's in the Doc correctly?

Reply:
Ah I wrote "Changed to 88px" for that one when I didn't. I DMed this one with Keegan to ask what his goal was, he was trying to align the width of the whole container with the bonus/episodes container, and the element-switcher. The 88px doesnt matter and this is the [fix we settled on](https://github.com/shift72/core-template/pull/464/files#diff-b9add1ee54c4a860fd60c71e1c243453ea9f48c98a3dd2596777b2239f6436cdR201)